index.js

```diff
@@ -52,6 +52,7 @@ module.exports = class Combobo {
     this.isHovering = false;
     this.autoFilter = this.config.autoFilter;
     this.optionsWithEventHandlers = new Set();
+    this.optionsWithKeyEventHandlers = new Set();
```

Comment:
why does there need to be a different set here? Shouldn't `optionsWithEventHandlers` cover ALL events?

Reply:
we are clearing ** optionsWithEventHandlers** whenever we clear all filters and it deals with options in the dropdown. And ** optionsWithKeyEventHandlers** will have all input fields and will be useful to prevent registering the key event multiple times. In application after calling setOptions() function, we are calling initEvents() function that is causing to register the key events more than once. In order to prevent it we are using **optionsWithKeyEventHandlers**